src/directives/clickOutside.ts

```diff
@@ -0,0 +1,21 @@
+export default function clickOutside(
```

Comment:
`directives`? I like that it's not "utils" or something like that, but perhaps we need to make it clear this is web-specific, or put inside the components directory?

Reply:
Directives is what svelte calls these functions 
https://svelte.dev/tutorial/actions

Maybe I'll rename the folder to `svelte-directives`?